app/models/task.rb

```diff
@@ -4,4 +4,40 @@ class Task < ActiveRecord::Base
   belongs_to :project
   acts_as_taggable
   validates_presence_of :title, :describe, :executor, :assigner
+  scope :state, -> (word){ where('state LIKE :w', w: "%#{word}%") if word.present?}
```

Comment:
Line is too long. [83/80]<br>Space missing to the left of {.<br>Space missing inside }.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.